<?xml version="1.0" ?>
<launch>
    <node pkg="nasa_robot_teleop" type="interactive_control.py" name="interactive_control" output="screen"> 
		<param name="robot" value="r2c_full"/>
		<param name="planner_type" value="moveit"/>
		<param name="group_config_file" value="$(find nasa_robot_teleop)/config/r2c_full.yaml"/>
		<param name="planner_config_file" value="$(find r2c_full_moveit_config)/config/r2c_full.srdf"/>
    </node>
</launch>